src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -20,15 +22,27 @@ interface DotEditPageNavItem {
 })
 export class DotEditPageNavComponent implements OnInit {
     @Input() pageState: DotRenderedPageState;
-
+    enterpriselicense: boolean;
     model: Observable<DotEditPageNavItem[]>;
 
-    constructor(public route: ActivatedRoute, private dotMessageService: DotMessageService) {}
+    constructor(public route: ActivatedRoute, private dotMessageService: DotMessageService, private dotcmsConfig: DotcmsConfig) {}
 
     ngOnInit(): void {
         this.model = this.dotMessageService
-            .getMessages(['editpage.toolbar.nav.content', 'editpage.toolbar.nav.layout', 'editpage.toolbar.nav.code'])
+            .getMessages([
+                'editpage.toolbar.nav.content',
+                'editpage.toolbar.nav.layout',
+                'editpage.toolbar.nav.code',
+                'editpage.toolbar.nav.licenseTooltip'
+            ])
             .mergeMap(() => Observable.of(this.getNavItems(this.pageState)));
+
+        this.dotcmsConfig
+            .getConfig()
+            .take(1)
+            .subscribe((res: any) => {
+                this.enterpriselicense = res.license.level >= 200 || false;
```

Comment:
This isn't a good practice, this can happen after line 74 happen, you can be 100% sure.

Also having a class public property to set something inside another class property is not a good idea.

Why don't you just merge the request with the get message and pass the license boolean to the ```getNavItems``` along with the page state?